### DISCUSSION TITLE
Possibilità to convert transaction massive

### DISCUSSION THREAD

**Discussion by michelebossa** [firefly-iii/firefly-iii#5548] [Firefly III] [answered]
Hi,
I'd like to convert a list of about 100 transaction type withdrawal to transfer i can by editing one by one but I can't do this in massive mode there is some way to do this?

Thanks

**Comment by JC5** [comment, marked as answer]
If you make a rule that matches all 100 transactions it could be done in a second.

**Reply by michelebossa** [reply]
I have a doubt about the trigger i see only for new or modify transaction how will work?

**Reply by JC5** [reply]
You can fire rules with a button.

**Reply by michelebossa** [reply]
I have tryed just now and it is worked thanks for the help
